x-pack/plugins/security_solution/server/search_strategy/security_solution/factory/network/top_n_flow/__mocks__/index.ts

```diff
@@ -118,6 +118,7 @@ export const mockSearchStrategyResponse: IEsSearchResponse<unknown> = {
                       _id: 'dd4fa2d4bd-1526378075029582',
                       _score: 0,
                       fields: {
+                        '@timestamp': ['2022-07-18T15:08:48.064Z'],
```

Comment:
the bug surfaced because some returned geo data included the timestamp. Adding this field here would break tests in `main` and proves the fix